_dev/src/views/product-feed-page.vue

```diff
@@ -57,11 +57,16 @@ export default {
       await this.$store.dispatch('googleAds/GET_GOOGLE_ADS_ACCOUNT');
       await this.$store.dispatch('smartShoppingCampaigns/GET_SSC_LIST');
     },
+    async connectAccounts() {
+      await this.$store.dispatch('accounts/REQUEST_GOOGLE_ACCOUNT_DETAILS');
+      await this.$store.dispatch('productFeed/GET_PRODUCT_FEED_SETTINGS');
+    },
```

Comment:
We're going to have the same method 3 times in the codebase. Having an action that dispatches the 2 other ones would be more interesting?

Reply:
TODO in next PR